### ISSUE TITLE
Cannot pass `...` to `ParamSpec` of `Callable`

### ISSUE THREAD

**Issue by bryanforbes** [microsoft/pyright#2716]
**Describe the bug**
Given the following code:

```python
from __future__ import annotations

from typing import Callable, Protocol
from typing_extensions import ParamSpec, TypeAlias

P = ParamSpec('P')

Handler: TypeAlias = Callable[P, None]

list_of_handlers: list[Handler[...]] = []


class HandlerProtocol(Protocol[P]):
    def __call__(self, /, *args: P.args, **kwargs: P.kwargs) -> None:
        ...


list_of_handler_protocols: list[HandlerProtocol[...]] = []
```

An error is reported for the `...` on line 10: `Expected ParamSpec or list of types` (the error reported for `list_of_handlers` is related to this error).

**Expected behavior**
No error should be reported and the type of `list_of_handlers` should be `list[Handler[(*args: Any, **kwargs: Any)]]`. This is the same behavior as seen for `list_of_handler_protocols`.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool?
Command-line

Which version?
1.1.196

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.197, which I just published. It will also be included in the next release of pylance.
